examples/lib/core.rego

```diff
@@ -31,7 +31,11 @@ name = resource.metadata.name
 kind = resource.kind
 labels = resource.metadata.labels
 annotations = resource.metadata.annotations
-
+gv := split(apiVersion, "/")
+group := gv[0] {
+    contains(apiVersion, "/")
```

Comment:
I think it would be good to set group to `core` if the apiVersion does not contain a slash, this way `core.group` is never undefined. This would be in line with the K8s documentation https://kubernetes.io/docs/reference/using-api/#api-groups

Reply:
ok, that makes sense